qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -101,6 +104,16 @@ def num_qubits(self) -> int:
         """
         return self.generators.num_qubits
 
+    def n_body(self, n: int) -> PauliLindbladError:
```

Comment:
feels to me like there's a verb missing for something that's not an attribute

```suggestion
    def restrict_num_bodies(self, num_bodies: int) -> PauliLindbladError:
```